network/load-balancer/quickstart.mdx

```diff
@@ -6,78 +6,86 @@ content:
   h1: Load Balancers - Quickstart
   paragraph: This page shows you how to get started with Scaleway Load Balancers.
 steps:
-  - step: TODO
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/network/load-balancer/quickstart
 dates:
   validation: 2022-11-03
   posted: 2021-05-26
 ---
 
 [Load Balancers](https://www.scaleway.com/en/load-balancer/) are highly available and fully-managed Instances which allow you to distribute your workload across your various servers. They ensure the scaling of all your applications while securing their continuous availability, even in the event of heavy traffic. Load Balancers are commonly used to improve the performance and reliability of web sites, applications, databases and other services.
 
+<Lightbox src="scaleway-load-balancer.webp" alt="" />
```

Comment:
I don't think this screenshot is appropriate since it's a screenshot from a Kapsule LB with a big red warning in it. Not really sexy :)

Reply:
No it's still there, just as above!
If you want, you could do a `yarn clean && yarn && yarn build && yarn serve`, which takes longer than a yarn develop, but builds the whole site more thoroughly. Once it's run, go to `http://localhost:9000/en/docs/` to view the build of the entire site, and that might solve the problel you're having with images.